java/ql/lib/ext/org.owasp.esapi.model.yml

```diff
@@ -3,4 +3,5 @@ extensions:
       pack: codeql/java-all
       extensible: summaryModel
     data:
-      - ["org.owasp.esapi", "Encoder", true, "encodeForHTML", "(String)", "", "Argument[0]", "ReturnValue", "taint", "manual"]
\ No newline at end of file
+      - ["org.owasp.esapi", "Encoder", true, "encodeForHTML", "(String)", "", "Argument[0]", "ReturnValue", "taint", "manual"]
+      - ["org.owasp.esapi", "Encoder", true, "encodeForBase64", "(byte[],boolean)", "", "Argument[0]", "ReturnValue", "taint", "manual"]
```

Comment:
We need to be careful with this summary — we may need to add it as a sanitizer in other queries (e.g. SQLi).

Reply:
Good point. `encodeForHTML` is already in `main`, so I'll leave that for now. But I have moved most of the modeling of `Encoder` methods into the query's dataflow configuration.